cms/models.py

```diff
@@ -257,6 +258,27 @@ class InfoLinks(Orderable):
     ]
 
 
+class SemesterDate(Orderable):
+    """
+    Dates for future start dates of semesters
+    """
+    program_page = ParentalKey(ProgramPage, related_name='semester_dates')
+    semester = models.CharField(
+        max_length=50,
+        help_text='Name for the semester, example: Fall or Fall 2017'
```

Comment:
Nitpick: can we make this help text always refer to a future date? I'd suggest doing the following:

```python
from datetime import datetime
now = datetime.now()

class SemesterDate(Orderable):
    ...
    semester_name = models.CharField(
        max_length=50,
        help_text='Name for the semester. For example: "Fall" or "Fall {year}"'.format(
            year=now.year + 1
        )
    )
```